src/main/groovy/com/google/protobuf/gradle/ProtobufExtension.groovy

```diff
@@ -54,13 +54,16 @@ abstract class ProtobufExtension {
    * "${project.buildDir}/generated/source/proto".
    */
   private String generatedFilesBaseDir
```

Comment:
I think also it's better to add a Deprecated annotation to the generateFilesBaseDir setter/getter.

Reply:
I agree adding Deprecated would be fair. The main problem is I wouldn't expect anyone to see it. I thought that we'd have to add a log statement or such if the setter is called.